.github/workflows/build-test.yml

```diff
@@ -52,13 +52,40 @@ jobs:
         . ./activate
         maturin build --no-sdist -i python --release
 
-    - name: Build Linux with maturin on Python ${{ matrix.python }}
+#    - name: Build Linux with maturin on Python ${{ matrix.python }}
+#      if: startsWith(matrix.os, 'ubuntu')
+#      run: |
+#        python -m venv venv
+#        ln -s venv/bin/activate
+#        . ./activate
+#        maturin build -i python --release --manylinux 2010
+
+    - name: Build Linux in manylinux2010 with maturin on Python ${{ matrix.python }}
       if: startsWith(matrix.os, 'ubuntu')
       run: |
-        python -m venv venv
-        ln -s venv/bin/activate
-        . ./activate
-        maturin build -i python --release --manylinux 2010
+        podman run --rm=true \
+          -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
+          quay.io/pypa/manylinux2010_x86_64 \
+          bash -exc '\
+            curl -L https://sh.rustup.rs > rustup.rs && \
+            chmod +x rustup.rs && \
+            ./rustup.rs -y && \
+            export PATH=$HOME/.cargo/bin:$PATH && \
+            rm -rf venv && \
+            PY_VERSION=${{ matrix.python }}
+            PY_VERSION=${PY_VERSION/.} && \
+            echo "Python version with dot removed is $PY_VERSION" && \
+            if [ "$PY_VERSION" = "37" ]; \
+            then export SCND_VERSION="${PY_VERSION}m"; \
```

Comment:
What does the `m` mean at the end of the `3.7` version?

Reply:
This is the piece of code I'm most grumpy about being forced to write in the last week due to Pypa... 

From the manylinux2010 documentation:
![image](https://user-images.githubusercontent.com/30377676/107911729-f4602180-6f11-11eb-920f-8cca32be09c4.png)